### DISCUSSION TITLE
✨ Change the uYouPlusExtra tweak name? - IMPORTANT POLL [Poll Ends on December 15th 2023]

### DISCUSSION THREAD

**Discussion by arichornlover** [arichornlover/uYouEnhanced#236] [Polls]
Hello everyone! I’ve seen some reddit posts related to this tweak and people didn’t seem to like this tweak being called uYouPlusExtra so I decided to do a poll on the new name vote below!
also if you don’t like any of the names then go for a Custom name by voting for it and then replying the new tweak name.

**Comment by ego-lay-atman-bay** [comment]
I don't think uYouPlus would be a good idea, because the original tweak made by qnblackcat is called uYouPlus, so that would raise a lot of confusion. Plus, it would probably not help with the situation at all.

**Reply by arichornlover** [reply]
Good call. I have removed it from the Poll.

**Comment by yodaluca23** [comment]
uYou++ is too basic, come on guys, like every other cracked app is "[App Name]++"

**Reply by arichornlover** [reply]
I got rid of it. I realized that would be far worse then uYouPlusExtra.

**Comment by columbusux** [comment]
yt-improve ( youtube improve )
uy-improve ( uYou improve )
yt-advanced ( youtube advanced )
uy-advanced ( uYou advanced )

and in my opinion we need new name makes it unique and different from the old and boring ones like (++/Plus) because sometimes it looks annoying and there is no need to add it. It is enough to have a completely different name that distinguishes you from everyone else so that when they mention you, they understand that it is you or the tweak itself that is being talked about.

For example, the name I would like is (**youtout**)
 is similar to YouTube, outside the framework or out of the ordinary, which is specifically out of the usual pattern, such as bein - beout.

**Reply by arichornlover** [reply]
I’ll reset the poll. And change it up. Thanks to the suggestion, I thought keeping the uYouPlus title wouldn’t be confusing since uYouPlusExtra is a fork to uYouPlus.

**Comment by arichornlover** [comment]
POLL has been reset! Thanks to columbusux & yodaluca23 
I have changed up the names to give them more unique titles compared to uYouPlus/uYou+.
but I did keep some names with uYou+ just in case.

**Comment by StNiosem** [comment]
UYou2 maybe?

**Comment by JBsitekr** [comment]
Bettertube

**Comment by machineonamission** [comment]
uYouPlusPlus is a cute one, like C++, also works well typed cause it can be shortened to uYou++

**Comment by PythonicNinja** [comment]
uTubeEnhanced

**Comment by PythonicNinja** [comment]
YouTubeEnhanced

**Reply by arichornlover** [reply]
That name won! But not for this repository.
it’s in a different repo I have. https://github.com/arichornlover/YouTubeEnhanced (old name: YouTubePlus)

**Comment by suhailusmani** [comment]
Enhanced-Tube

**Comment by favna** [comment]
Voted for uYou+2 because it also sounds kinda funny but if you want to be very cheeky then you do uYouVanced or something like that 🤣

**Comment by iGerman00** [comment]
uYouToo or uYouTwo. Nice wordplay, easy to write, no special characters or numbers unless you write it as uYou2

**Reply by arichornlover** [reply]
cannot be called uYou2.
and there’s a reason

calling it uYou2 will be VERY confusing because people who follow this repository would think that a new tweak called uYou2 released (which doesn’t even exist), and the naming can be a problem with this.  and I may potentially consequences from tweak devs when using this name. Hope that sums it up.

**Comment by KieranBLK** [comment]
uYouEnhanced it good but by the looks of it, it seems you've already decided? 😅

![image](https://github.com/arichornlover/uYouEnhanced/assets/120525685/97ab6563-cbcd-4613-9fc8-4e0c774f3193)

**Reply by arichornlover** [reply]
Well yeah, I’m just doing some preparations for it since it feels like it might be the new tweak name 😅

**Reply by arichornlover** [reply]
Also I already made the url to work with the upcoming new name 😉
https://github.com/arichornlover/uYouEnhanced

**Comment by olivertzeng** [comment]
uYTube

**Comment by baljeethakur** [comment]
iTube+

**Comment by Proaxel** [comment]
I'm going with uYouPlus2, I think it best conveys to clueless noobs that this is a direct continuation of the uYouPlus project

**Comment by JudeTheTechSavvyDude** [comment]
I think it should be named UYouUltimate for sure definitely

**Reply by JudeTheTechSavvyDude** [reply]
I made an edit to the Name i think it should definitely be named UYouUltimate instead of uYouPlusUltra what do u think of this

**Comment by preland** [comment]
OurTube 😉

Y’know, since it’s just YouTube, but with what we collectively want from the app.

**Comment by Trewer07** [comment]
uYouPhoenix

![IMG_0793](https://github.com/arichorn/uYouPlusExtra/assets/153033123/b0323fd2-dfde-4ea8-ac84-4eede046038a)

**Comment by GhostIsBeHere** [comment]
As funny as something like uYouPlusProExtra+ would be, uYouEnhanced is better

**Comment by zahid-58** [comment]
uYou2b or uYou2be
uYou2+ or uYu or uY2be

**Comment by ZXXII** [comment]
More suggestions:
- uYouUltimate
- uYouPro

**Comment by alx-xlx** [comment]
Yes the "uYouEnhanced" is much better. Anything not including "plus", "+", "Extra" is what I think should be the criteria.

But I also think that, the previous name gave the idea of the where it was forked from and its sources, so there should also be a graph/tree diagram of some sort showing the the other versions of the same app. and should be on the README.

Renaming won't change the fact that a new user would still be confused as to which version suits

If anyone can guide me the the correct article that shows this info I can create a diagram.

**Comment by ZXXII** [comment]
Another suggestion:
uYouExtra 

(just remove the Plus)

**Comment by SAM-AEL** [comment]
iYouTube / UTube+ / YouPlusTube

**Comment by RealWitki** [comment]
In my opinion the best way is to create new, simple name that loosely relates to uYouPlus, so something like "uYu" albo "uYoo"

On the other hand it's sounds and looks like uwu...

**Comment by markcst** [comment]
uTube

**Comment by NebulaBC** [comment]
I voted for uYouEnhanced, but after sitting here for a bit, I think that uYou+2 might be the route to go. I've seen the +2 naming scheme with other software, and it works out well. It rolls off the tongue really nice.

**Reply by arichornlover** [reply]
If it somehow wins whenever the poll ends on Dec 15th then I consider changing it to uYou+2 but right now the most voted name is uYouEnhanced, but there’s still 1 week left in the poll so there’s a chance that it can change.

**Comment by Dr-Sauce** [comment]
YOUnified

**Comment by arichornlover** [comment]
Pull ended a week early, because I knew the poll was gonna stay on uYouEnhanced. But anyways good job guys!
